admin/sql/updates/schema-change/27.mirror_only.sql

```diff
@@ -0,0 +1,124 @@
+ALTER TRIGGER apply_artist_release_pending_updates_slave ON track RENAME TO apply_artist_release_pending_updates_mirror;
+
+DROP TRIGGER IF EXISTS a_ins_release_slave ON release;
+DROP TRIGGER IF EXISTS a_upd_release_slave ON release;
+DROP TRIGGER IF EXISTS a_del_release_slave ON release;
+DROP TRIGGER IF EXISTS a_ins_release_event_slave ON release_country;
+DROP TRIGGER IF EXISTS a_upd_release_event_slave ON release_country;
+DROP TRIGGER IF EXISTS a_del_release_event_slave ON release_country;
+DROP TRIGGER IF EXISTS a_ins_release_group_slave ON release_group;
+DROP TRIGGER IF EXISTS a_upd_release_group_slave ON release_group;
+DROP TRIGGER IF EXISTS a_del_release_group_slave ON release_group;
+DROP TRIGGER IF EXISTS a_upd_release_group_meta_slave ON release_group_meta;
+DROP TRIGGER IF EXISTS a_ins_release_group_secondary_type_join_slave ON release_group_secondary_type_join;
+DROP TRIGGER IF EXISTS a_del_release_group_secondary_type_join_slave ON release_group_secondary_type_join;
+DROP TRIGGER IF EXISTS a_ins_release_label_slave ON release_label;
+DROP TRIGGER IF EXISTS a_upd_release_label_slave ON release_label;
+DROP TRIGGER IF EXISTS a_del_release_label_slave ON release_label;
+DROP TRIGGER IF EXISTS a_ins_track_slave ON track;
+DROP TRIGGER IF EXISTS a_upd_track_slave ON track;
+DROP TRIGGER IF EXISTS a_del_track_slave ON track;
+
+ALTER FUNCTION a_ins_release_slave() RENAME TO a_ins_release_mirror;
+ALTER FUNCTION a_upd_release_slave() RENAME TO a_upd_release_mirror;
+ALTER FUNCTION a_del_release_slave() RENAME TO a_del_release_mirror;
+ALTER FUNCTION a_ins_release_event_slave() RENAME TO a_ins_release_event_mirror;
+ALTER FUNCTION a_upd_release_event_slave() RENAME TO a_upd_release_event_mirror;
+ALTER FUNCTION a_del_release_event_slave() RENAME TO a_del_release_event_mirror;
+ALTER FUNCTION a_ins_release_group_slave() RENAME TO a_ins_release_group_mirror;
+ALTER FUNCTION a_upd_release_group_slave() RENAME TO a_upd_release_group_mirror;
+ALTER FUNCTION a_del_release_group_slave() RENAME TO a_del_release_group_mirror;
+ALTER FUNCTION a_upd_release_group_meta_slave() RENAME TO a_upd_release_group_meta_mirror;
+ALTER FUNCTION a_ins_release_group_secondary_type_join_slave() RENAME TO a_ins_release_group_secondary_type_join_mirror;
+ALTER FUNCTION a_del_release_group_secondary_type_join_slave() RENAME TO a_del_release_group_secondary_type_join_mirror;
+ALTER FUNCTION a_ins_release_label_slave() RENAME TO a_ins_release_label_mirror;
+ALTER FUNCTION a_upd_release_label_slave() RENAME TO a_upd_release_label_mirror;
+ALTER FUNCTION a_del_release_label_slave() RENAME TO a_del_release_label_mirror;
+ALTER FUNCTION a_ins_track_slave() RENAME TO a_ins_track_mirror;
+ALTER FUNCTION a_upd_track_slave() RENAME TO a_upd_track_mirror;
+ALTER FUNCTION a_del_track_slave() RENAME TO a_del_track_mirror;
+
+
+CREATE TRIGGER a_ins_release_mirror AFTER INSERT ON release
```

Comment:
This needs to be regenerated for the updates you made (but also regardless given the invalid `\\set`).